lib/order_list.rb

```diff
@@ -0,0 +1,54 @@
+require_relative 'dish_list'
+require_relative 'personal_details'
+require 'twilio-ruby'
+
+class OrderList
+
+  extend Forwardable
+
+  def_delegator :@dish_list, :price_list
+
+  attr_reader :client
+
+  def initialize(dish_list)
+    @dish_list = dish_list
+    account_sid = PersonalDetails::AccountSid
+    auth_token = PersonalDetails::AuthToken
+    @client = Twilio::REST::Client.new(account_sid, auth_token)
+  end
+
+  def place_order(amount_list,total=nil)
+    fail 'An ordered dish is unavailable' unless
+        ordered_dishes_available?(amount_list)
+    fail 'Total does not match sum of dishes in order' unless
+       total.nil? || total == dish_sum(amount_list)
+    @client.messages.create(
+    from: PersonalDetails::TwiioPhone,
+    to: PersonalDetails::PersonalPhone,
+    body:"Thank you! Your order was placed and will be delivered before #{Time.new.hour+1}:"+ Time.new.strftime("%M")
```

Comment:
Metrics/LineLength: Line is too long. [117/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)